core/scss/components/select/select.scss

```diff
@@ -92,10 +92,42 @@
         border-radius: 0 0 4px 4px;
       }
     }
+
   }
   .#{$select__class}__dropdown {
     background: $md-white-100;
     border-radius: $event-overlay__border-radius;
     box-shadow: $event-overlay__border, $event-overlay__shadow;
+    display: flex;
+    flex-direction: column;
+
+    .md-input-group.md-search-input {
+      margin: rem-calc(12) rem-calc(12) 0 rem-calc(12);
+
+      .md-icon {
+        margin-left: 12px;
+        margin-right: 12px;
+      }
+
+      .md-input  {
+        padding-left: rem-calc(37);
```

Comment:
37px seems like an odd number.  Did we verify with design?

Reply:
The actual input and search icon match the 12px padding from the design when in use. Ran into some spacing issues when placing the input inside the cdk overlay and adjusted manually. Can take another look at it.